web/src/components/tasks/LicenseTask.tsx

```diff
@@ -87,19 +89,19 @@ export const LicenseTask = (props: Props): ReactElement => {
 
     setIsLoading(true);
     api
-      .checkLicenseStatus(nameAndAddress, props.task.id as LicenseTaskId)
+      .checkLicenseStatus(nameAndAddress)
       .then((result: UserData) => {
         const resultLicenseData =
           result.businesses[result.currentBusinessId]?.licenseData?.licenses?.[licenseNameForTask];
-
-        if (!resultLicenseData) return;
-        analytics.event.task_address_form.response.success_application_found();
```

Comment:
sand - you need to figure out if this function needs to be delete or placed in another area.